assets/src/scripts/components/map/nldiMapping.js

```diff
@@ -1,7 +1,12 @@
 import { geoJson, circleMarker } from 'leaflet';
-import { get } from '../../ajax';
-import config from '../../config';
-import { select } from 'd3-selection';
+import {
+    markerFillColor,
```

Comment:
I think these belong in here and then legend can import them. What do you think? I think that makes it clearer that these are nldi style settings.

Reply:
Yeah I think that's a better approach, I'll switch this around.